app/controllers/api/v1/admin/status_actions_controller.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class Api::V1::Admin::StatusActionsController < Api::BaseController
+  # modeled on api/v1/admin/account_actions_controller.rb
```

Comment:
I left 3 of these `modeled on` comments in so that it's clear where some of the new code is descended/remixed from. Happy to remove them before merging.